Python/fitnesse/html_util.py

```diff
@@ -2,41 +2,41 @@
 
 from fitnesse.context import SuiteResponder, PageCrawlerImpl, PageData, PathParser, WikiPage, WikiPagePath
 
-
+class TestType:
+    SETUP: "SetUp"
+    TEARDOWN: "TearDown"
 class HtmlUtil:
 
+    @staticmethod
+    def get_Path(page_name: str, wiki_page: WikiPage):
+        inherit_Page: WikiPage = page_crawler.get_inherited_page(page_name, wiki_page)
+        if inherit_Page is not None:
+            path: wiki_page.get_page_crawler().get_full_path(setup)
+            path_name: str = path_parser.render(page_path)
+            return path_name
```

Comment:
Please consider not returning NULL in case the "inherit_Page" is None (this condition has not been taken care of). Alternatively, you need to account for this scenario when using the function.